packages/typo3-guides-extension/src/Renderer/SinglePageRenderer.php

```diff
@@ -25,7 +25,7 @@ public function render(RenderCommand $renderCommand): void
         );
 
         $context->getDestination()->put(
-            $renderCommand->getDestinationPath() . '/index.html',
+            $renderCommand->getDestinationPath() . '/singlepage-index.html',
```

Comment:
In the current rendering the file is saved at singlehtml/Index.html

I would suggest, we do the same